src/Sylius/Bundle/CoreBundle/PaymentRequest/Processor/Offline/CaptureProcessor.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\PaymentRequest\Processor\Offline;
+
+use Sylius\Component\Payment\Model\PaymentInterface;
+use Sylius\Component\Payment\Model\PaymentRequestInterface;
+
+final class CaptureProcessor implements CaptureProcessorInterface
+{
+    public function process(PaymentRequestInterface $paymentRequest): void
+    {
+        $payment = $paymentRequest->getPayment();
+
+        $responseData = $payment->getDetails();
+        if (PaymentInterface::STATE_NEW === $payment->getState()) {
+            $responseData = [
+                'paid' => false,
+            ];
+        }
+
+        $paymentRequest->setResponseData($responseData);
+        $paymentRequest->setState(PaymentRequestInterface::STATE_COMPLETED);
```

Comment:
Right now it's not require, let's keep this idea for a future PR.

Reply:
maybe therefore we should change function to somethin like `$paymentRequest->complete()`?

Reply:
I'm afraid we are introducing a new concept here since all states have to get their own method and can be misunderstood : `->new()`, `->failed()`.
Payum have those kind of method : `->markNew()`, `->markFailed()`.
Rethinking it twice I don't know if we wanted those kind of methods.